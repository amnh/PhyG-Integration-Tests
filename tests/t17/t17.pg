--build(distance, rdwag), inputting tnt files
set(seed:1634561640)
read(tnt:"ordered.ss")
read(tnt:"small.ss")
read(tnt:"test0.ss")
read(tnt:"test1.ss")
read(tnt:"test01.ss")
build(distance, rdWag, best:10)
report("t17.csv", data, overwrite)
report("t17_cr.csv", crossrefs, overwrite)
report("t17_ia.txt", ia, concatenate, includemissing, overwrite)
report("t17.tre", newick, overwrite)