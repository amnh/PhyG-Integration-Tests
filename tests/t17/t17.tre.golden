(A:0.0,(((d:0.0,f:0.0)HTU30:0.0,(i:0.0,((g:0.0,((E:1.0,((two:0.0,(D:0.0,b:0.0)HTU35:0.0)HTU29:0.0,(B:0.0,c:0.0)HTU32:0.0)HTU23:0.0)HTU33:0.0,(one:0.0,((C:0.0,four:0.0)HTU27:0.0,(e:0.0,six:0.0)HTU34:0.0)HTU22:1.0)HTU36:0.0)HTU28:0.0)HTU25:1.0,(h:0.0,three:0.0)HTU26:0.0)HTU21:0.0)HTU37:0.0)HTU24:0.0,(a:0.0,five:0.0)HTU31:0.0)HTU38:0.0)HTU20[36.0];
(A:0.0,((E:1.0,((D:0.0,three:0.0)HTU29:0.0,(d:0.0,two:0.0)HTU31:1.0)HTU22:0.0)HTU28:0.0,((((e:0.0,(c:0.0,h:0.0)HTU30:0.0)HTU26:0.0,(six:0.0,((b:0.0,i:0.0)HTU27:0.0,(B:0.0,one:0.0)HTU33:0.0)HTU23:0.0)HTU32:0.0)HTU21:1.0,(a:0.0,five:0.0)HTU35:0.0)HTU25:0.0,(four:0.0,(f:0.0,(C:0.0,g:0.0)HTU34:0.0)HTU24:0.0)HTU36:0.0)HTU37:0.0)HTU38:0.0)HTU20[36.0];
(A:0.0,(((six:0.0,((b:1.0,((f:0.0,((E:0.0,d:0.0)HTU26:0.0,(g:0.0,h:0.0)HTU33:0.0)HTU23:0.0)HTU29:2.0,(two:1.0,(C:0.0,(c:0.0,five:0.0)HTU34:0.0)HTU25:0.0)HTU37:0.0)HTU21:0.0)HTU27:0.0,(three:0.0,(D:0.0,(a:0.0,four:0.0)HTU35:0.0)HTU24:0.0)HTU36:0.0)HTU28:0.0)HTU30:0.0,(i:0.0,one:0.0)HTU32:0.0)HTU22:0.0,(B:0.0,e:0.0)HTU31:0.0)HTU38:0.0)HTU20[36.0];
(A:0.0,(six:0.0,(((f:0.0,((h:0.0,((c:0.0,four:0.0)HTU29:0.0,(C:0.0,three:0.0)HTU35:0.0)HTU22:0.0)HTU33:0.0,((b:0.0,g:0.0)HTU26:0.0,(five:0.0,i:0.0)HTU27:0.0)HTU34:0.0)HTU21:0.0)HTU25:0.0,(two:0.0,((one:0.0,(B:0.0,e:0.0)HTU36:0.0)HTU28:0.0,(E:0.0,a:0.0)HTU30:1.0)HTU24:0.0)HTU32:0.0)HTU23:0.0,(D:0.0,d:0.0)HTU37:0.0)HTU31:0.0)HTU38:0.0)HTU20[35.0];
(A:0.0,(f:0.0,((((c:0.0,((D:0.0,e:0.0)HTU29:0.0,(a:0.0,six:0.0)HTU31:0.0)HTU23:0.0)HTU30:0.0,(i:0.0,((((E:0.0,five:0.0)HTU26:0.0,(four:0.0,three:0.0)HTU28:0.0)HTU27:0.0,(one:0.0,two:0.0)HTU33:0.0)HTU25:0.0,(b:0.0,g:0.0)HTU37:0.0)HTU21:0.0)HTU35:1.0)HTU24:0.0,(C:0.0,h:0.0)HTU36:0.0)HTU32:0.0,(B:0.0,d:0.0)HTU34:0.0)HTU22:0.0)HTU38:0.0)HTU20[34.0];
(A:0.0,(f:0.0,((d:0.0,((C:0.0,(e:0.0,five:0.0)HTU35:0.0)HTU26:0.0,(three:0.0,((b:0.0,four:0.0)HTU30:0.0,(h:0.0,((D:0.0,(one:0.0,two:0.0)HTU33:0.0)HTU25:0.0,(E:0.0,i:0.0)HTU31:1.0)HTU24:0.0)HTU37:1.0)HTU21:0.0)HTU29:0.0)HTU27:0.0)HTU28:0.0,(c:0.0,((B:0.0,g:0.0)HTU32:0.0,(a:0.0,six:0.0)HTU36:0.0)HTU23:0.0)HTU34:0.0)HTU22:0.0)HTU38:0.0)HTU20[34.0];
(A:0.0,(((E:0.0,three:0.0)HTU28:1.0,(h:0.0,((c:0.0,((one:0.0,two:0.0)HTU27:0.0,(i:0.0,((D:0.0,a:0.0)HTU26:0.0,(b:0.0,g:0.0)HTU35:0.0)HTU23:0.0)HTU37:0.0)HTU21:0.0)HTU25:1.0,(B:0.0,((C:0.0,six:0.0)HTU31:0.0,(e:0.0,five:0.0)HTU33:0.0)HTU24:0.0)HTU34:0.0)HTU36:0.0)HTU30:0.0)HTU22:0.0,(four:0.0,(d:0.0,f:0.0)HTU32:0.0)HTU29:0.0)HTU38:0.0)HTU20[33.0];
(A:0.0,(three:0.0,(four:0.0,((E:0.0,f:0.0)HTU28:1.0,(b:1.0,((e:0.0,((h:0.0,two:0.0)HTU25:0.0,(i:0.0,(B:0.0,(c:0.0,one:0.0)HTU33:0.0)HTU22:0.0)HTU37:0.0)HTU31:0.0)HTU26:2.0,(a:1.0,((six:0.0,(D:0.0,d:0.0)HTU36:0.0)HTU30:0.0,(g:0.0,(C:0.0,five:0.0)HTU35:0.0)HTU34:0.0)HTU24:0.0)HTU32:0.0)HTU21:0.0)HTU29:0.0)HTU23:0.0)HTU27:0.0)HTU38:0.0)HTU20[33.0];
(A:0.0,(f:0.0,(six:0.0,((B:0.0,a:0.0)HTU33:0.0,(d:0.0,((h:0.0,((one:0.0,two:0.0)HTU29:1.0,(b:1.0,((four:0.0,(c:0.0,e:0.0)HTU36:0.0)HTU25:0.0,(E:0.0,three:0.0)HTU27:0.0)HTU37:0.0)HTU31:0.0)HTU21:0.0)HTU32:1.0,(i:0.0,((D:0.0,g:0.0)HTU28:0.0,(C:0.0,five:0.0)HTU30:0.0)HTU24:0.0)HTU35:0.0)HTU22:1.0)HTU34:0.0)HTU23:0.0)HTU26:0.0)HTU38:0.0)HTU20[33.0];
(A:0.0,(b:0.0,(two:0.0,(c:0.0,((D:0.0,a:0.0)HTU25:0.0,(i:0.0,((E:0.0,(d:0.0,f:0.0)HTU34:0.0)HTU23:1.0,(one:0.0,((e:0.0,g:0.0)HTU27:0.0,((four:0.0,three:0.0)HTU30:0.0,((C:0.0,(B:0.0,five:0.0)HTU37:0.0)HTU32:0.0,(h:0.0,six:0.0)HTU35:0.0)HTU31:0.0)HTU28:0.0)HTU22:1.0)HTU33:0.0)HTU29:0.0)HTU26:0.0)HTU21:0.0)HTU36:0.0)HTU24:0.0)HTU38:0.0)HTU20[33.0];
